lib/tapioca/ruby_ext/forking_patch.rb

```diff
@@ -0,0 +1,42 @@
+# typed: strict
+# frozen_string_literal: true
+
+module Forking
```

Comment:
These should not be in the `ruby_ext` folder, since they are actually internal modules of `Isolation`. Can we move these back please?